chimedb/dataset/insert.py

```diff
@@ -78,7 +78,7 @@ def insert_dataset(ds_id, base_dset, is_root, state, time):
     DatasetState.DoesNotExist
         If the state attached to the dataset is not found in the database.
     """
```

Comment:
Your change looks good. However, I'm wondering what the code would do if the base `Dataset` did not exist. It looks like it would just raise a `Dataset.DoesNotExist` exception from L86. Do we want to report that more explicitly, or at least document it in the `Raises` section of the docstring above.

Reply:
Good catch. Yeah, the archiver expects this to throw in that case, it just wasn't documented.